models/CountryCode.js

```diff
@@ -0,0 +1,37 @@
+const { Model, DataTypes } = require('sequelize');
+const sequelize = require("../config/connection");
+
+class CountryCode extends Model {
+
+}
+
+CountryCode.init(
+    {
+        id: {
+            type: DataTypes.INTEGER,
+            allowNull: false,
+            primaryKey: true,
+            autoIncrement: true
+        },
+        country_name: {
```

Comment:
suggest re-naming to "country_code_name" to help differentiate from Location model

Reply:
Renamed as suggested